Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Graham Campbell <graham@mineuk.com>
+ */
+class PhpdocScalarFixerTest extends AbstractFixerTestBase
+{
+    public function testBasicFix()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @return int
+     */
+
+EOF;
+
+        $input = <<<'EOF'
+<?php
+    /**
+     * @return integer
+     */
+
+EOF;
+
+        $this->makeTest($expected, $input);
+    }
+
+    public function testFixWithTabsOnOneLine()
+    {
+        $expected = "<?php /**\t@return\tbool\t*/";
+
+        $input = "<?php /**\t@return\tboolean\t*/";
+
+        $this->makeTest($expected, $input);
+    }
+
+    public function testFixMoreThings()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * Hello there mr integer!
+     *
+     * @param int|float
+     *
+     * @return string|bool
+     */
+
+EOF;
+
+        $input = <<<'EOF'
+<?php
+    /**
+     * Hello there mr integer!
+     *
+     * @param integer|real
+     *
+     * @return string|boolean
+     */
+
+EOF;
+
+        $this->makeTest($expected, $input);
+    }
+
+    public function testFixVar()
+    {
+        $expected = <<<'EOF'
+<?php
+    /**
+     * @var bool|int|Double Booleans, integers and doubles.
+     */
+
+EOF;
+
+        $input = <<<'EOF'
+<?php
+    /**
+     * @var boolean|integer|Double Booleans, integers and doubles.
```

Comment:
What if it would be
` @var integer Some integer value.`
? Would the only first word will be changed (as the second is part of sentence, not the typehinting part) ?